tlair/src/test/scala/at/forsyte/apalache/tla/lir/IrGenerators.scala

```diff
@@ -85,7 +94,9 @@ trait IrGenerators {
    */
   def genBool: Gen[ValEx] = for {
     b <- arbitrary[Boolean]
-  } yield ValEx(TlaBool(b))
+    i <- arbitrary[Int]
+    tt <- oneOf(Untyped(), Typed[Int](i))
```

Comment:
Should this maybe be factored out into an `arbitraryType` constructor or something?

Reply:
That makes sense